packages/eslint-plugin/tests/rules/use-lifecycle-interface/cases.ts

```diff
@@ -120,6 +138,19 @@ export const invalid = [
         },
       },
     ],
+    annotatedOutput: `
+        @Injectable()
+        class Test implements DoBootstrap {
```

Comment:
This didn't fix all the issues?

Reply:
If I understood the documentation of eslint correctly the fixes are applied again and again, as long as the lint error is still reported. I'm unsure, how I can use the plugin in a local testing repo to test that out, please advice me on that.

Reply:
So yes I found a way to try it out. (run e2e tests and then try your changes in tmp/e2e-fixtures/ in any of the apps there. Maybe that should be documented) Indeed eslint runs again on the output of fixes until there are no changes, so it indeed results in a fix that adds all the implements clauses.

Reply:
Only thing I noticed is that I do not add an import for the Interface, here I would like guidance @JamesHenry

Reply:
@bulldog98 Sorry for the delay. We have a utility for it used in a few other rules, it's very straightforward: https://github.com/angular-eslint/angular-eslint/pull/1829

Reply:
I want to get things ready for Angular 18, so I went ahead and applied the change